.github/workflows/native-build-development.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         run: |
           mvn -B clean install --fail-at-end -Pnative -Ddocker \
             -Dquarkus.native.container-build=true \
-            -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java8 \
+            -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:20.1.0-java8 \
```

Comment:
Shouldn't we switch to the java11 flavor for this and the installed JDK? We made it the default.

Reply:
I initially thought that the point of this cron was to run the quickstarts with Java 8, but now I see that that is not the case.